### PR TITLE
(Doc+) troubleshoot ILM videos

### PR DESCRIPTION
**Pull request by stefnestor**
👋🏽 howdy, team! This links to my + @anniegale9538 's six newest [Support Troubleshooting videos](https://www.youtube.com/playlist?list=PL_mJOmq4zsHbQlfEMEh_30_LuV_hZp-3d) which are about resolving general ILM Health & the top five ILM rollover errors to the existing [Troubleshooting ILM errors](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-lifecycle-error-handling.html). It side quests to link the watermark error to [its troubleshooting doc](https://www.elastic.co/guide/en/elasticsearch/reference/master/fix-watermark-errors.html).

### CODE REVIEW

**Comment by leemthompo** [comment]
@stefnestor my [comment here](https://github.com/elastic/elasticsearch/pull/114915#discussion_r1803254400) applies to all these links too 🙏

File: docs/reference/ilm/error-handling.asciidoc [hunk 1]
```diff
@@ -8,6 +8,9 @@ When this happens, {ilm-init} moves the index to an `ERROR` step.
 If {ilm-init} cannot resolve the error automatically, execution is halted  
 until you resolve the underlying issues with the policy, index, or cluster.
 
+See link:https://www.youtube.com/watch?v=VCIqkji3IwY[this video]
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:11]:
```suggestion
See this https://www.youtube.com/watch?v=VCIqkji3IwY[{ilm-init} health video] 
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 2]
```diff
@@ -8,6 +8,9 @@ When this happens, {ilm-init} moves the index to an `ERROR` step.
 If {ilm-init} cannot resolve the error automatically, execution is halted  
 until you resolve the underlying issues with the policy, index, or cluster.
 
+See link:https://www.youtube.com/watch?v=VCIqkji3IwY[this video] 
+for example troubleshooting walkthrough of {ilm-init} health.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:12]:
```suggestion
for example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 3]
```diff
@@ -183,6 +186,8 @@ The rollover action then manages setting and updating the alias to
 
 Do not explicitly configure this same alias in the aliases section of an index template.
 
+See link:https://www.youtube.com/watch?v=Ww5POq4zZtY[this video] for example troubleshooting walkthrough.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:189]:
```suggestion
See this https://www.youtube.com/watch?v=Ww5POq4zZtY[resolving `duplicate alias` video] for example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 4]
```diff
@@ -191,20 +196,26 @@ Either the index is using the wrong alias or the alias does not exist.
 Check the `index.lifecycle.rollover_alias` <<indices-get-settings, index setting>>.
 To see what aliases are configured, use <<cat-alias, _cat/aliases>>.
 
+See link:https://www.youtube.com/watch?v=NKSe67x7aw8[this video] for example troubleshooting walkthrough.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:199]:
```suggestion
See this https://www.youtube.com/watch?v=NKSe67x7aw8[resolving `not point to index` video] for an example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 5]
```diff
@@ -191,20 +196,26 @@ Either the index is using the wrong alias or the alias does not exist.
...
+
 [discrete]
 ==== Setting [index.lifecycle.rollover_alias] for index [y] is empty or not defined
 
 The `index.lifecycle.rollover_alias` setting must be configured for the rollover action to work.
 
 Update the index settings to set `index.lifecycle.rollover_alias`.
 
+See link:https://www.youtube.com/watch?v=LRpMC2GS_FQ[this video] for example troubleshooting walkthrough.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:208]:
```suggestion
See this https://www.youtube.com/watch?v=LRpMC2GS_FQ[resolving `empty or not defined` video] for an example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 6]
```diff
@@ -191,20 +196,26 @@ Either the index is using the wrong alias or the alias does not exist.
...
+
 [discrete]
 ==== Alias [x] has more than one write index [y,z]
 
 Only one index can be designated as the write index for a particular alias.
 
 Use the <<indices-aliases, aliases>> API to set `is_write_index:false` for all but one index.
 
+See link:https://www.youtube.com/watch?v=jCUvZCT5Hm4[this video] for example troubleshooting walkthrough.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:217]:
```suggestion
See this https://www.youtube.com/watch?v=jCUvZCT5Hm4[resolving `more than one write index` video] for an example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 7]
```diff
@@ -214,6 +225,8 @@ For example, `my-index` does not match the pattern requirement.
 
 Append a numeric value to the index name, for example `my-index-000001`.
 
+See link:https://www.youtube.com/watch?v=9sp1zF6iL00[this video] for example troubleshooting walkthrough.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:228]:
```suggestion
See this https://www.youtube.com/watch?v=9sp1zF6iL00[resolving `does not match pattern` video] for an example troubleshooting walkthrough.
```

File: docs/reference/ilm/error-handling.asciidoc [hunk 8]
```diff
@@ -227,8 +240,7 @@ For more information, see <<circuit-breaker-errors>>.
 
 This indicates that the cluster is running out of disk space.
 This can happen when you don't have {ilm} set up to roll over from hot to warm nodes.
-
-Consider adding nodes, upgrading your hardware, or deleting unneeded indices.
+For more information, see <<fix-watermark-errors>>.
```
Comment (stefnestor) [inline docs/reference/ilm/error-handling.asciidoc:243]:
```suggestion
For more information, see <<fix-watermark-errors,Fix watermark errors>>.
```

**Comment by stefnestor** [comment]
Thanks, @leemthompo ! Updated to match your requested formatting 🙏

**Review by leemthompo — APPROVED** [review]
LGTM. Thanks for following up!

**Comment by leemthompo** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/114932)|
|✅|8.16|[](https://github.com/elastic/elasticsearch/pull/114933)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**